src/macros.rs

```diff
@@ -150,3 +150,24 @@ macro_rules! wrap_pymodule {
         }
     };
 }
+
+/// Add current module to the existing table of built-in modules.
+///
+/// Use it before [`prepare_freethreaded_python`](crate::prepare_freethreaded_python) and
+/// leave feature `auto-initialize` off
+#[macro_export]
```

Comment:
I don't think `PyImport_AppendInittab` is implemented by PyPy.

```suggestion
#[macro_export]
#[cfg(not(PyPy))]
```

Reply:
Added above

Reply:
👍, CI won't rerun until merge conflict is fixed.